docs/resources/slo_correction.md

```diff
@@ -0,0 +1,66 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_slo_correction Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Resource for interacting with the slo_correction API
+---
+
+# datadog_slo_correction (Resource)
+
+Resource for interacting with the slo_correction API
+
+## Example Usage
+
+```terraform
+# Create a new Datadog SLO Correction. slo_id can be derived from slo resource or specify an slo id of an existing SLO.
+
+resource "datadog_service_level_objective" "example_slo" {
+  name        = "example slo"
+  type        = "metric"
+  description = "some updated description about example_slo SLO"
+  query {
+    numerator   = "sum:my.metric{type:good}.as_count()"
+    denominator = "sum:my.metric{type:good}.as_count() + sum:my.metric{type:bad}.as_count()"
+  }
+
+  thresholds {
+    timeframe = "7d"
+    target    = 99.5
+    warning   = 99.8
+  }
+  tags = ["foo:bar"]
+}
+resource "datadog_slo_correction" "example_slo_correction" {
+  category    = "Scheduled Maintenance"
+  description = "correction example"
+  end         = 1735718600
```

Comment:
I wonder if  for documentation there is a way to add a comment next to timestamp (like human readable value)

Reply:
do we need comment here in the terraform example as we already have description for start/end in epoch seconds?